package.json

```diff
@@ -54,7 +54,7 @@
     "qs": "^6.9.4",
     "re-resizable": "^6.5.4",
     "react": "^17.0.2",
-    "react-bootstrap": "^1.0.1",
+    "react-bootstrap": "^1.6.1",
```

Comment:
A bit unclear, it may have been Yurii split several PRs of a same working branch, yet I think the bootstrap update had already landed https://github.com/sparkletown/sparkle/pull/2010#issuecomment-902607506

Reply:
ah, it's an old comment. 🙈 i have reached Yurii in slack about that and forgot to remove it from here. Thanks, Goran. 🙃 
resolving the conversation.